.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-       - image: circleci/ruby:latest
+       - image: circleci/python:3.6
```

Comment:
might be worth considering switching this to the new python image in ecr kenneth just added